app/views/tag/show/_tab_content.html.erb

```diff
@@ -1,7 +1,8 @@
 <div class="tab-content" style="clear:both;">
 
   <% if @start && @end %>
-    <p><%= raw translation('tag.show.displaying_content') %> <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %>. <a href="/tag/<%=h params[:id] %>">View all</a></p>
+  <p><%= raw translation('tag.show.displaying_content') %> <%= @start.to_formatted_s(:long) %> to <%= @end.to_formatted_s(:long) %>. <a href="/tag/<%=h params[:id] %>"><%= raw translation('tag.show.view_all') %></a></p>
```

Comment:
Hi @RishiKumarRay, can you indent line 4 properly? Thank you!

The rest is perfect.

Reply:
Hi, @TildaDares I am not able to make a commit and open up a pull request for this indentation. I tried doing so